repose-aggregator/src/docs/asciidoc/services/http-connection-pool.adoc

```diff
@@ -1,6 +1,183 @@
+     Data will be sent earlier, at the cost of an increase in bandwidth consumption. +
+     Default is true.
+<13> If a Keep-Alive header is not present in the response, the value of keepalive.timeout is evaluated.
+     If this value is 0, the connection will be kept alive indefinitely.
+     If the value is greater than 0, the connection will be kept alive for the number of milliseconds specified.
+     Set to 1 to connect:close.
+     Default is 0.
+<14> The keystore filename either fully qualified or relative to the Repose configuration directory (/etc/repose is the default).
+     Used for SSL/TLS Client Authentication. +
+     NOTE: The `keystore-filename`, `keystore-password`, and `key-password` are all mutually required. If one exists, then they all must exist and client authentication is assumed to be required.
+<15> The password to access the keystore. +
+     NOTE: The `keystore-filename`, `keystore-password`, and `key-password` are all mutually required. If one exists, then they all must exist and client authentication is assumed to be required.
+<16> The password for the particular key within the keystore. +
+     NOTE: The `keystore-filename`, `keystore-password`, and `key-password` are all mutually required. If one exists, then they all must exist and client authentication is assumed to be required.
+<17> The truststore filename either fully qualified or relative to the Repose configuration directory (/etc/repose is the default).
+     Used when it is necessary to have the certificate of the requested resource's server. +
+     NOTE: The `truststore-filename` and `truststore-password` are mutually required. If one exists, then the other must exist.
+<18> The password to access the truststore. +
+     NOTE: The `truststore-filename` and `truststore-password` are mutually required. If one exists, then the other must exist.
+<19> Indicates that the configured list of headers should be added to every outbound request on this connection pool.
+     Use is optional.
+<20> A header to be added to every outbound request from this connection pool.
+     At least on is required when using the optional `headers` element.
+<21> A second pool is defined here that uses all the defaults.
+
+== Common Use Cases/Problems
+
+=== Origin Service doesn't support chunked encoding (e.g. WSGI)
+
+If the origin service doesn't support chunked encoding, it can be turned off easily.
+Add or update the `chunked-encoding` attribute for your default pool to false.
+
+[source,xml]
+.partial http-connection-pool.cfg
+----
+<pool id="default"
+      default="true" <!--1-->
+      ... <!--2-->
+      chunked-encoding="false"/> <!--3-->
+</pool>
+----
+<1> Be sure it's the default pool being updated.
+<2> Here's the rest of the pool attributes.
+<3> Add `chunked-encoding` if not already present and set it to false.
+
+[NOTE]
```

Comment:
The admonition block notation isn't used here. Did we always want to use that notation, or just when necessary? The block notation would be:
```
[NOTE]
====
po0l? po0l po0l po0l po0l po0l. po0l!
====
```